tests/api/administrator.contact.1.0.0.soap.Cest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_weblinks
+ *
+ * @copyright   Copyright (C) 2005 - 2015 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+class AdministratorContacts1SoapCest
+{
+	/**
+	 * Set up the contact stub
+	 */
+	public function __construct()
+	{
+		$this->faker = Faker\Factory::create();
+		$this->name = $this->faker->bothify('AdministratorContacts1SoapCest contact ?##?');
+		$this->category = 4; // 4 is by default the id of Uncategorised category
+		$this->id = 'this property will contain the id of the created contact';
+	}
+
+	/*
+	public function existsWsdl(ApiTester $I)
+	{
+		$I->comment('I execute a request to ensure that redCORE automatically generates the .wsdl schema on the fly');
+		$I->sendGET('index.php?webserviceClient=administrator&webserviceVersion=1.0.0&option=contact&api=soap&wsdl');
+		sleep(1);
+		$I->sendGET('media/redcore/webservices/joomla/administrator.contact.1.0.0.wsdl');
+		$I->seeResponseCodeIs(200);
+		$I->seeResponseIsXml();
+	}
```

Comment:
It was created already but I'm leaving this commented code for the future just in case.

Is the one who takes care of generating it.